src/mediaEditor/plugins/mediaAlignment/mediaAlignmentComponent.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021  (original work) Open Assessment Technologies SA;
+ *
+ * @author Juan Luis Gutierrez Dos Santos <juanluis.gutierrezdossantos@taotesting.com>
+ */
+
+/**
+ * Controls media size
+ */
+ import $ from 'jquery';
+ import _ from 'lodash';
+ import component from 'ui/component';
+ import tpl from 'ui/mediaEditor/plugins/mediaAlignment/tpl/mediaAlignment';
+ import 'ui/mediaEditor/plugins/mediaAlignment/style.css';
+ import { FLOAT_LEFT_CLASS, FLOAT_RIGHT_CLASS } from './helper';
+ 
+ /**
+  * Creates mediaAlignment component
+  * @param $container
+  * @param media
+  * @param config
```

Comment:
This `config` param is described in the comment but does not exist in the component